src/styles/shop.css

```
@@ -1,37 +1,22 @@
+ * This file is injected as a plain <link> stylesheet (imported with `?url`),
+ * so it never passes through the Tailwind pipeline. The `bg-shop-panel`,
+ * `text-shop-muted`, `font-shop-display` utilities are generated from the
+ * matching `@theme inline` block in `app.css`; here we only declare the raw
+ * CSS variables those tokens resolve to, plus the light/dark swap and the
+ * keyframes used by the live-stock indicator and CTA button.
  *
  * Everything else is Tailwind utilities on components — no bespoke class
  * selectors, no global leakage.
  */
 
 @import url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..900;1,9..40,100..900&family=DM+Mono:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..900;1,9..40,100..900&family=DM+Mono:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=DM+Sans:ital,opsz,wght@0,9..40,100..900;1,9..40,100..900&family=DM+Mono:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&display=swap'" (import-notation)

(import-notation)